pkg/manifest/link.go

```diff
@@ -296,6 +296,18 @@ func (ll LinkList) AllAreVideo() bool {
 	return true
 }
 
+// Returns whether all the resources in the collection are bitmaps or video clips.
+func (ll LinkList) AllAreVisual() bool {
+	for _, link := range ll {
+		mt := link.MediaType()
+		if !mt.IsBitmap() && !mt.IsVideo() {
+			return false
+		}
+		// TODO should we check alternates?
```

Comment:
I kept this TODO as it is a good question. We don't support alternates at the moment so `false` is reasonable, but that might change.